### PR TITLE
docs: provide information on local user locks from login attempts

### PR DESCRIPTION
**Pull request by stevenGravy**

### CODE REVIEW

File: docs/pages/reference/authentication.mdx [hunk 1]
```diff
@@ -121,6 +121,49 @@ $ tctl create -f cap.yaml
...
+A lock on attempting logins is placed for 20 minutes if within a 30 minute window 
+a local user has:
+- 5 failed login attempts or
+- 3 failed password resets
+
+After the lock has expired the user may attempt to login again.
+
+Overriding the lock is available to users with rights to maintain `user` resources,
+available in the builtin `editor` role.  To turn off a lock update the user entry
+to disable the lock. Here are steps to complete that.
```
Comment (alexfornuto) [inline docs/pages/reference/authentication.mdx:135]: Suggested copy edits.

```suggestion
A lock is placed on a local user if, within a 30 minute window, a local user has:

- 5 failed login attempts or
- 3 failed password resets

The lock lasts 20 minutes. After the lock has expired the user may attempt to login again.

Overriding a lock is available to users with rights to maintain `user` resources,
available in the builtin `editor` role. To turn off a lock, update the user entry
to disable the lock, following these steps.
```

File: docs/pages/reference/authentication.mdx [hunk 2]
```diff
@@ -121,6 +121,49 @@ $ tctl create -f cap.yaml
...
+
+Retrieve the user entry so you can edit the status:
+
+```code
+$ tctl get users/jeff > jeff.yaml`
```
Comment (alexfornuto) [inline docs/pages/reference/authentication.mdx:140]: Can/should we use `tctl edit` here?

File: docs/pages/reference/authentication.mdx [hunk 3]
```diff
@@ -121,6 +121,49 @@ $ tctl create -f cap.yaml
 </TabItem>
 </Tabs>
 
+### Local User Locks
```
Comment (ptgott) [inline docs/pages/reference/authentication.mdx:124]: Per the style guide, we are using sentence case for headings. This should be "Local user locks"

File: docs/pages/reference/authentication.mdx (hunk 2 above)
Comment (zmb3) [reply docs/pages/reference/authentication.mdx:140]: For v13 and v12 we can.

Comment (stevenGravy) [reply docs/pages/reference/authentication.mdx:140]: When I last tried it wasn't allowing it in v12. Is there an update coming?

Comment (zmb3) [reply docs/pages/reference/authentication.mdx:140]: No, `tctl edit` has just only been supported for v12+

Comment (stevenGravy) [reply docs/pages/reference/authentication.mdx:140]: This issue #21598

Comment (zmb3) [reply docs/pages/reference/authentication.mdx:140]: Ah, right.

Comment (stevenGravy) [reply docs/pages/reference/authentication.mdx:140]: Let's go ahead with this and can update if that changes

File: docs/pages/reference/authentication.mdx [hunk 4]
```diff
@@ -121,6 +121,49 @@ $ tctl create -f cap.yaml
...
 </Tabs>
 
+### Local user locks
+
+A lock is placed on a local user if, within a 30 minute window, a local user has:
+
+- 5 failed login attempts or
+- 3 failed password resets
+
+The lock lasts 20 minutes. After the lock has expired the user may attempt to login again.
```
Comment (ptgott) [inline docs/pages/reference/authentication.mdx:131]:
```suggestion
The lock lasts 20 minutes. After the lock has expired the user may attempt to log in again.
```
"Login" is a noun.

File: docs/pages/reference/authentication.mdx [hunk 5]
```diff
@@ -121,6 +121,49 @@ $ tctl create -f cap.yaml
...
+
+Overriding a lock is available to users with rights to maintain `user` resources,
+available in the builtin `editor` role. To turn off a lock, update the user entry
```
Comment (ptgott) [inline docs/pages/reference/authentication.mdx:134]:
```suggestion
available in the built-in `editor` role. To turn off a lock, update the user entry
```

File: docs/pages/reference/authentication.mdx [hunk 6]
```diff
@@ -121,6 +121,49 @@ $ tctl create -f cap.yaml
...
+  - access
+  status:
+    is_locked: true
+    lock_expires: "2023-04-22T01:55:02.228158166Z"
+    locked_message: user has exceeded maximum failed login attempts
+version: v2
+```
+
+Update the `is_locked` under `status` to `false` and save the file. Now
+update the user entry with the below command.
```
Comment (ptgott) [inline docs/pages/reference/authentication.mdx:159]:
```suggestion
update the user entry with the command below:
```

File: docs/pages/reference/authentication.mdx [hunk 7]
```diff
@@ -121,6 +121,49 @@ $ tctl create -f cap.yaml
...
+  roles:
+  - access
+  status:
+    is_locked: true
+    lock_expires: "2023-04-22T01:55:02.228158166Z"
+    locked_message: user has exceeded maximum failed login attempts
+version: v2
+```
+
+Update the `is_locked` under `status` to `false` and save the file. Now
```
Comment (ptgott) [inline docs/pages/reference/authentication.mdx:158]:
```suggestion
Update the `is_locked` field under `status` to `false` and save the file. Now
```

**Review by ptgott — APPROVED** [review]

File: docs/pages/reference/authentication.mdx [hunk 8]
```diff
@@ -121,6 +121,50 @@ $ tctl create -f cap.yaml
 </TabItem>
 </Tabs>
 
+### Local user locks
+
+A lock is placed on a local user if, within a 30 minute window, a local user has:
```
Comment (zmb3) [inline docs/pages/reference/authentication.mdx:126]: Should we call this something else? It sounds a lot like we're referring to Teleport's lock resource, but this is something else.

File: docs/pages/reference/authentication.mdx [hunk 9]
```diff
@@ -121,6 +121,50 @@ $ tctl create -f cap.yaml
...
+
+- 5 failed login attempts or
```
Comment (zmb3) [inline docs/pages/reference/authentication.mdx:128]: It's really easy for this to become out of sync with the code. I wonder if we need to mention the specifics or if we can just say that teleport will lock accounts after too many failed logins or password resets.

Comment (stevenGravy) [reply docs/pages/reference/authentication.mdx]: Please see update

File: docs/pages/reference/authentication.mdx (hunk 8 above)
Comment (stevenGravy) [reply docs/pages/reference/authentication.mdx]: Please see update

**Comment by stevenGravy** [comment]
@zmb3 please take a look when you can on the updates.

**Review by r0mant — APPROVED** [review]

**Pull request merged**